assets/pages/monthly-contributions/components/paymentMethods.jsx

```diff
@@ -15,22 +16,29 @@ type PropTypes = {
   email: string,
   firstName: string,
   lastName: string,
+  error: ?string,
 };
 
 
 // ----- Component ----- //
 
 function PaymentMethods(props: PropTypes) {
 
-  let content = 'Please fill in all the fields above.';
+  let errorMessage = '';
+  let stripeButton = <StripePopUpButton email={props.email} callback={postCheckout} />;
+  if ((props.firstName === '' || props.lastName === '') && props.error === null) {
```

Comment:
The logic prioritises the fact that a serious error took place between support-frontend's backend and the step functions and we show that error. 

In order to see that error, the user should have filled the fields correctly and submit the form in the first place.